utilities/EluvioCli.js

```diff
@@ -731,16 +849,16 @@ yargs(hideBin(process.argv))
   )
 
   .command(
-    "account_set_tenant <tenant_admins>",
-    "Sets the tenant admins group address for this account.",
+    "account_set_tenant <tenant>",
+    "Sets the tenant contract to assoicate with this account - if an admin group id is used, it will be converted to the tenant contract id that belongs to the group.",
```

Comment:
`Associate this account to the tenant - if an admins group ID is used, it will be converted to the tenant ID.`